### PR TITLE
Github actions config for Integration tests

### PR DESCRIPTION
**Pull request by adheeeem**

### CODE REVIEW

File: src/MRA.Jobs.Api/Infrastructure/ConfigureServices.cs [hunk 1]
```diff
@@ -25,22 +25,18 @@ public static class ConfigureServices
         //services.AddAzureEmailService();//uncomment this if u wont use email service from Azure from namespace Mra.Shared.Initializer.Azure.EmailService
 
         services.AddScoped<AuditableEntitySaveChangesInterceptor>();
-        string dbConnectionString = configuration.GetConnectionString("SqlServer");
 
-        bool useInMemoryDatabase = Environment.GetEnvironmentVariable("USE_IN_MEMORY_DATABASE") == "true";
+        string dbConnectionString = configuration.GetConnectionString("SqlServer");
+        var cl = configuration["UseInMemoryDatabase"];
```
Comment (komdil) [inline src/MRA.Jobs.Api/Infrastructure/ConfigureServices.cs:30]: you can remove this

File: src/MRA.Jobs.Api/Infrastructure/dbsettings.Development.json
Comment (komdil) [inline src/MRA.Jobs.Api/Infrastructure/dbsettings.Development.json:1]: I don't think that you need this file here

File: src/MRA.Jobs.Api/Infrastructure/dbsettings.json
Comment (komdil) [inline src/MRA.Jobs.Api/Infrastructure/dbsettings.json:1]: I don't think that you need this file here

File: src/MRA.Jobs.Api/WebUI/appsettings.json [hunk 2]
```diff
@@ -1,4 +1,8 @@
 {
+  "JwtSettings": {
```
Comment (komdil) [inline src/MRA.Jobs.Api/WebUI/appsettings.json:2]: It should not be here. In production we will get it from azure key vault

File: tests/MRA.Jobs.Api/Application.IntegrationTests/appsettings.json
Comment (komdil) [inline tests/MRA.Jobs.Api/Application.IntegrationTests/appsettings.json:1]: I don't think that you need this file here

**Review by komdil — CHANGES_REQUESTED** [review]

File: src/MRA.Jobs.Api/WebUI/appsettings.json (hunk 2 above)
Comment (komdil) [reply src/MRA.Jobs.Api/WebUI/appsettings.json:2]: You need to get it from appsettings.Development.json

**Review by komdil — CHANGES_REQUESTED** [review]

File: src/MRA.Jobs.Api/WebUI/appsettings.json (hunk 2 above)
Comment (adheeeem) [reply src/MRA.Jobs.Api/WebUI/appsettings.json:2]: Ok

**Review by komdil — APPROVED** [review]

**Review by Muhammadislom99 — APPROVED** [review]

**Pull request merged**